boot/freeldr/freeldr/bootmgr.c

```diff
@@ -58,8 +58,10 @@ static const struct
     {"Drive"       , EditCustomBootDisk      , LoadAndBootDevice},
     {"Partition"   , EditCustomBootPartition , LoadAndBootDevice},
     {"BootSector"  , EditCustomBootSectorFile, LoadAndBootDevice},
```

Comment:
I also doubt that those three methods (drive, partition and bootsector) are also available for the moment in UEFIBOOT :)